pkgs/development/tools/misc/help2man/default.nix

```diff
@@ -17,7 +17,11 @@ stdenv.mkDerivation rec {
   strictDeps = true;
 
   nativeBuildInputs = [ gettext perlPackages.perl perlPackages.LocaleGettext ];
-  buildInputs = [ perlPackages.LocaleGettext ];
+  buildInputs = [ perlPackages.LocaleGettext libintl ];
```

Comment:
It's a little coupled. The `--enable-nls` check requires that `bindtextdomain` be available. If enabled, it changes the perl script to use `Locale::gettext`, and also generates an `LD_PRELOAD` wrapper that intercepts `bindtextdomain`. 

I think in this case that `libintl` is correct, since it's the runtime component that is required and IIUC these are provided by glibc. So only non-glibc systems would need a library for `bindtextdomain`.

Reply:
`libintl = if stdenv.hostPlatform.libc != "glibc" then gettext else null;`
means that libintl is gettext so lets just use gettext directly without an alias

Reply:
I was assuming that it should be `null` on linux (well, glibc systems) and use the glibc versions. I don't have a source for this assumption though.